### PR TITLE
Fix security issues

### PR DESCRIPTION
**Pull request by obiPlabon**

### CODE REVIEW

**Review by kowsar89 — CHANGES_REQUESTED** [review]
template version not updated

File: includes/classes/class-user.php [hunk 1]
```diff
@@ -1,264 +1,293 @@
 <?php
 /**
- * User class of Directorist
+ * Directorist user class.
  *
- * This class is for interacting database table
+ * User registration and acess management related functionalities can be found in this class.
```
Comment (kowsar89) [inline includes/classes/class-user.php:5]: Typo

**Review by kowsar89 — CHANGES_REQUESTED** [review]

File: includes/classes/class-user.php (hunk 1 above)
Comment (obiPlabon) [reply includes/classes/class-user.php]: Fixed

**Review by kowsar89 — APPROVED** [review]
Works fine

**Review by vairafiq — APPROVED** [review]

**Pull request merged**